css/component_styles/snackbar.css

```diff
@@ -0,0 +1,30 @@
+.snackbar {
+    background-color: var(--snackbar-color);
+    border-radius: 5px;
+    display: flex;
+    align-items: center;
+    padding: 1rem;
+    width: 60%;
+    margin: auto;
+    box-shadow: var(--snackbar-shadow);
+}
+
+.snackbar-text {
+    margin-right: auto;
```

Comment:
not sure if this is really required(everything is left-aligned until and unless you add some other style)

Reply:
actually I did this to push the button to the other side as Vikram suggested to use margin-left/right auto instead of space-between. So I did that only .